### ISSUE TITLE
Uncaught TypeError: Cannot read property 'forEach' of null

### ISSUE THREAD

**Issue by ielgnaw** [akalajzi/atom-file-bookmark#14]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [file-bookmark](https://github.com/akalajzi/atom-file-bookmark) package, v1.0.2
### Stack Trace

Uncaught TypeError: Cannot read property 'forEach' of null

```
At /Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:204

TypeError: Cannot read property 'forEach' of null
    at FileBookmark._handleGitStatus (/Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:204:11)
    at FileBookmark._handleGitStatus (/Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:1:1)
    at FileBookmark.add (/Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:147:8)
    at FileBookmark.toggleBookmark (/Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:136:62)
    at HTMLSpanElement.<anonymous> (/Users/ielgnaw/.atom/packages/file-bookmark/lib/file-bookmark.coffee:88:68)
    at HTMLDivElement.jQuery.event.dispatch (/Users/ielgnaw/.atom/packages/file-bookmark/node_modules/jquery/dist/jquery.js:5110:27)
    at HTMLDivElement.elemData.handle (/Users/ielgnaw/.atom/packages/file-bookmark/node_modules/jquery/dist/jquery.js:4918:28)
```
### Commands

```
     -0:52.5.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -0:52.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.9.0 project-manager:list-projects (atom-text-editor.editor.is-focused)
     -0:31 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:30.1.0 project-manager:alt-confirm (atom-text-editor.editor.mini.is-focused)
     -0:25.6.0 project-manager:list-projects (atom-text-editor.editor.is-focused)
     -0:24.3.0 project-manager:alt-confirm (atom-text-editor.editor.mini.is-focused)
     -0:20.2.0 file-bookmark:toggle-panel (atom-text-editor.editor.is-focused)
     -0:12.3.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:12.3.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:08.9.0 project-manager:list-projects (atom-text-editor.editor.is-focused)
     -0:08.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:07.6.0 project-manager:alt-confirm (atom-text-editor.editor.mini.is-focused)
  2x -0:03.9.0 file-bookmark:toggle-panel (atom-text-editor.editor.is-focused)
     -0:02.0 pane:show-item-1 (atom-text-editor.editor.is-focused)
     -0:01.2.0 core:close (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "atom-commander"
    ],
    "themes": [
      "one-dark-ui",
      "monokai"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-commander, v0.7.1 (inactive)
autoclose-html, v0.23.0 (active)
block-comment, v0.4.3 (active)
docblockr, v0.8.5 (active)
emmet, v2.4.3 (active)
file-bookmark, v1.0.2 (active)
file-icons, v1.7.19 (active)
highlight-selected, v0.11.2 (active)
language-babel, v2.37.1 (active)
minimap, v4.25.0 (active)
minimap-highlight-selected, v4.4.0 (active)
monokai, v0.18.0 (active)
open-in-browser, v0.4.7 (inactive)
open-terminal-here, v2.1.4 (inactive)
project-manager, v3.2.0 (active)
Stylus, v3.0.0 (active)
Sublime-Style-Column-Selection, v1.7.2 (active)
svg-preview, v0.9.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by soundslikeinfo** [comment]
I received this error after toggling open the Bookmarked files panel for the first time after opening a project window.

**Comment by akalajzi** [comment]
Hopefully fixed with 1.0.4 patch
